1.9/administration/installing/cloud/gce.md

```diff
@@ -267,7 +267,7 @@ Do not change this parameter. It is required by the Google Cloud SDK.
 Specify the filename for the DC/OS installer. Default `dcos_generate_config.sh`.
 
 ### dcos_installer_download_path
-Specify the location of where the DC/OS installer is available from [dcos.io](https://dcos.io). Default: `https://downloads.dcos.io/dcos/testing/master/{{ dcos_installer_filename }}?_ga=1.3100023.674486911.1475259096`. The value of `{{ dcos_installer_file }}` is described above.
+Specify the location of where the DC/OS installer is available from [dcos.io](https://dcos.io). Default: `https://downloads.dcos.io/dcos/stable/{{ dcos_installer_filename }}`. The value of `{{ dcos_installer_file }}` is described above.
```

Comment:
Not sure sure if it's a sha but it's required for google analytics. When I originally wrote the GCE installation I got the URL from the DCOS website. For installing dcos on GCE the _ga url parameter isn't required.